src/bot/storage/actual.ts

```
@@ -0,0 +1,254 @@
+    }
+
+    try {
+      if (transactionsByActualAccountId.size > 0) {
+        await this.sendTransactionsToActual(
+          transactionsByActualAccountId,
+          stats,
+          onProgress,
+        );
+      }
+    } catch (error) {
+      throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the catch clause.


(lint/complexity/noUselessCatch)